test/unittest/capability/caldera/caldera_test.go

```diff
@@ -49,6 +49,17 @@ func TestExecuteB64(t *testing.T) {
 	assert.Equal(t, cacao.NewVariables(), results)
 }
 
+func TestParseYamlAbility(t *testing.T) {
+	resultingAbility := caldera.ParseYamlAbility([]byte("ability_id: 9a30740d-3aa8-4c23-8efa-d51215e8a5b9"))
```

Comment:
Note `ability_id:`, not `id:` in the YAML as per the [spec example](https://docs.oasis-open.org/cacao/security-playbooks/v2.0/cs01/security-playbooks-v2.0-cs01.html#_Toc152256493). That's a bit sad. Doing it this way, we skip a complicated YAML -> JSON -> Ability chain, so I would prefer this. Other attributes except for the ID seem to match up, perhaps we can find a way to dynamically fix specifically the ID.

Reply:
(I'd rather **not** patch the API model, as it's a very bad :tm: idea to patch autogenerated code)